### DISCUSSION TITLE
adding explicit variable in Qfield app

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3221] [Q&A]
How can I have explicit variable attached in Qfield app integrated as one of the attributes in layer

Example : If I add additional variable :  name of the person in the qfield app . Then how do I retrieve this as attribute to layer. Is there a way out?  Where can I use the variables already present in Qfield app and how.

My basic idea is to get the name of the person as an attribute without he actually inputting it in the attribute list.
Can this be possible in postgis layer

**Comment by suricactus** [comment]
Basically duplicate of https://github.com/opengisch/QField/discussions/3207
